imap_processing/tests/ialirt/unit/test_process_ephemeris.py

```diff
@@ -138,3 +138,23 @@ def test_build_output(furnish_kernels):
                     time_interval,
                 )
             )
+
+
+@pytest.mark.external_kernel
+def test_generate_text_files(furnish_kernels):
+    kernels = [
+        "naif0012.tls",
+        "pck00011.tpc",
+        "de440s.bsp",
+        "imap_spk_demo.bsp",
+        "earth_latest_high_prec.bpc",
+    ]
+    with furnish_kernels(kernels):
+        process_ephemeris.generate_text_files(
+            station="Kiel", day="20250813", file_path="../test_text_files"
+        )
+
+    with open("../test_text_files/20250813_Kiel.txt") as file:
```

Comment:
This should use the pytest tmpdir fixture, that way you don't get dangling files left around after a test potentially fails.

Reply:
Done, thanks for the suggestion!